### PR TITLE
NPM Run Chrome Ammended

### PR DESCRIPTION
**Pull request by architajain12**
Issue #96 fixed. Created a npm run chrome script.

### CODE REVIEW

File: chrome/run.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
```
Comment (miketaylr) [inline chrome/run.js]: Can you add a license header here? 

See https://github.com/webcompat/webcompat-reporter-extensions/blob/master/firefox/addon.js#L1, for example:

File: chrome/run.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+const chromeLaunch = require('chrome-launch'); // eslint-disable-line import/no-extraneous-dependencies
+const url = 'https://webcompat.com/'; // this can be any url, we can also pass through the command line
+const args = ['--load-extension=./dist/chrome'];
```
Comment (miketaylr) [inline chrome/run.js]: nit: can we get a new line between the variable declarations and calling `chromeLaunch`?

File: chrome/run.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env node
+const chromeLaunch = require('chrome-launch'); // eslint-disable-line import/no-extraneous-dependencies
```
Comment (miketaylr) [inline chrome/run.js]: I *don't* think the `no-extraneous-deps` comment is needed since chrome-launch is in our package.json. Does eslint throw an error if you remove it?

(you can test by running `npm run eslint`)

File: chrome/run.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+const url = 'https://webcompat.com/'; // this can be any url, we can also pass through the command line
+const args = ['--load-extension=./dist/chrome'];
+chromeLaunch(url, { args });
+console.log('A new instance of Chrome should now be open in the background.');
```
Comment (miketaylr) [inline chrome/run.js]: eslint will complain about `console.log`, but it seems valid to me. Can you add a comment to disable the error here?

**Review by miketaylr — CHANGES_REQUESTED** [review]
Wow, awesome!

Just a few minor changes, but this is totally working. Great job.

Be sure to run `npm run eslint` to catch the eslint errors, otherwise the Travis job will fail.

File: .eslintrc [hunk 5]
```diff
@@ -21,7 +21,7 @@
     "eqeqeq": [2, "smart"],
     "indent": [2, 2],
     "keyword-spacing": 2,
-    "linebreak-style": [2, "unix"],
+    "linebreak-style": [2, "windows"],
```
Comment (miketaylr) [inline .eslintrc]: @architajain12 what's the rationale for this change?

**Comment by miketaylr** [comment]
Thanks for the changes! Just one more question/tweak, then we're good to merge.

File: .eslintrc (hunk 5 above)
Comment (architajain12) [reply .eslintrc]: @miketaylr  I was using a windows pc and `npm run eslint` gave me linebreak errors so I had to change it to "windows". I'm reverting it to "unix" right now and sending you another PR.

**Comment by miketaylr** [comment]
Thank you! All tests passing, and verified this works.

**Review by miketaylr — APPROVED** [review]

File: package.json [hunk 6]
```diff
@@ -6,6 +6,7 @@
   "scripts": {
```
Comment (pqpq699) [inline package.json:6]: h

**Review by pqpq699 — COMMENTED** [review]
`gh[hh]() #61
